hnn_core/network.py

```diff
@@ -29,8 +29,8 @@ def read_spikes(fname, gid_ranges=None):
 
     Returns
     ----------
-    spikes : Spikes
-        An instance of the Spikes object.
+    spikes : CellResponse
```

Comment:
```suggestion
   cell_response : CellResponse
```

?

Reply:
I'll try it out. This rename will change quite a lot of the code, but we may as well keep the naming consistent until we think of a better alternative.